docs/CHANGELOG.asciidoc

```diff
@@ -96,6 +96,28 @@ For detailed information about the n.n.n release, review the following sections.
 
 The 7.11.2 release includes the following bug fixes. For information on the breaking changes, refer to <<breaking-changes-7.11,Breaking changes in 7.11>>.
 
+[[known-issues-7.11.2]]
+[discrete]
+=== Known issues
+
+When upgrading from 7.11.0 or 7.11.1 to 7.11.2, certain connectors, including those that connect to Jira,
+ServiceNow, and IBM Resilient, are not properly migrated during the upgrade process, causing them to be
+deleted.
+
+Impacts include:
+
+* Kibana Alerting and Action alerts that have been configured to use the affected connectors will no
```

Comment:
The UI feature is Alerts and Actions.  Also, we put UI feature names in bold.

Reply:
Noted! I went ahead and updated this.